storage/Devices/Bcache.h

```diff
@@ -78,6 +122,16 @@ namespace storage
 	 */
 	static std::vector<const Bcache*> get_all(const Devicegraph* devicegraph);
 
+	/**
+	 * Fix numbers of bcache devices, so there is no holes in numbers between in memory
```

Comment:
> Fix the numeric ids of bcache devices so that there are no holes in the id sequence for in-memory
bcache devices. Called automatically when a device is deleted and a hole can appear.